engine/src/platform-internal.h

```diff
@@ -369,6 +369,10 @@ void MCPlatformCallbackSendMenuSelect(MCPlatformMenuRef menu, uindex_t item);
 
 void MCPlatformCallbackSendViewFocusSwitched(MCPlatformWindowRef window, uint32_t view_id);
 
+#ifndef _PTHREAD_H
```

Comment:
You only need to include this in platform.cpp

Reply:
@runrevmark  done